bedrock/firefox/tests/test_helpers.py

```diff
@@ -545,28 +545,17 @@ def test_android_notes(self):
             "",
             {"newsletter_id": "download-firefox-mobile"},
         ),
+        # Android and iOS platforms should have the same newsletter_id since they share the same email template.
         (
             "platform='android'",
-            {"newsletter_id": "download-firefox-android"},
+            {"newsletter_id": "download-firefox-mobile"},
         ),
         (
             "platform='ios'",
-            {"newsletter_id": "download-firefox-ios"},
-        ),
-        (
-            "platform='all'",
             {"newsletter_id": "download-firefox-mobile"},
         ),
         (
-            "message_set='fx-android', platform='android'",
-            {"newsletter_id": "get-android-embed"},
```

Comment:
**question:** I notice some of the newsletter IDs being removed here are still in `newsletter-strings.json`. Is the intention to keep those so that people can still unsubscribe from them in the management page? Or could we remove them there also if the emails are in fact, no longer active?

Reply:
Is that why it seems like many of these are very old but we still keep them around? The ones I removed from newsletter-strings were not referenced anywhere else, just in that file and the tests, so it seemed safe to remove them.

Reply:
I'm not sure of the answer re: what is safe to remove and when. I guess if a newsletter is no longer active, it should be OK to remove as an option from the preference center?

Would you agree, @pmac ?

Reply:
If they're newsletters, even old ones, they could show up on people's Email Pref Center page as long as said newsletter is still setup in the basket database. If they're not in the Newsletters table on basket, then they can be removed.

Reply:
Or if they're really just transactional email IDs, those never show on the pref center either.